traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -814,7 +814,7 @@ func (dss *TODSSDeliveryService) Read(h http.Header, useIMS bool) ([]interface{}
 
 func selectMaxLastUpdatedQuery(where string) string {
 	return `SELECT max(t) from (
-		SELECT max(dss.last_updated) as t from deliveryservice_server dss ` + where +
+		SELECT max(dss.last_updated) as t from deliveryservice_server dss JOIN deliveryservice ds ON ds.id = dss.deliveryservice ` + where +
```

Comment:
We don't always need this join, only when dsId is a query parameter. How is the join conditionally added to `serverCountQuery` and `selectQuery`?

Reply:
We will actually need this one always, because we look for the DS ID in the params and if its not present, we return there. So if it makes it to this line, we are guaranteed that we do have a delivery service id, denoted by `id` in the params.

Reply:
I missed that this one was in a completely different file. Sounds good